opensearch/src/main/java/org/opensearch/sql/opensearch/storage/OpenSearchStorageEngine.java

```diff
@@ -33,4 +38,15 @@ public Table getTable(DataSourceSchemaName dataSourceSchemaName, String name) {
       return new OpenSearchIndex(client, settings, name);
     }
   }
+
+  @Override
+  public TableScanOperator getTableScan(String indexName, String scrollId) {
+    // TODO call `getTable` here?
```

Comment:
IS TODO needed? 
I don't know why you would need to get table for subsequent calls

Reply:
To get an `OpenSearchExprValueFactory`, otherwise we have to serialize it.